web/testing/commonHeader.js

```diff
@@ -98,16 +128,16 @@
     {
       case 1:
         sKbd=document.getElementById('kbd_id1').value;
-        kmw.addKeyboards(sKbd);
+        doAddKeyboards(sKbd);
         break;
       case 2:
         sKbd=document.getElementById('kbd_id2').value.toLowerCase();
-        kmw.addKeyboards('@'+sKbd);
+        doAddKeyboards('@'+sKbd);
         break;
       case 3:
         // Add keyboard for comma-separated language name(s)
         sKbd=document.getElementById('kbd_id3').value;
-        kmw.addKeyboardsForLanguage(sKbd);
+        doAddKeyboardsForLanguage(sKbd);
```

Comment:
Can the commonHeader.js change also be applied to
web/testing/promise-api/promisehdr.js ?

Reply:
This looks like it is already done in promisehdr.js? Slightly differently, but the same basic idea.